mock-up/scss/tech.scss

```
@@ -25,13 +26,20 @@
     border-radius: 5px;
     bottom: 10px;
     height: 350px;
-    right: 10px;
     position: absolute;
+    right: 10px;
     width: 225px;
     z-index: -1;
   }
 }
 
+.tech:hover {
+  &::after {
+    top: 10px;
+    left: 10px;
```

The transition is only for `right` and `bottom` properties.
Try this:
```suggestion
    right: -10px;
    bottom: -10px;
```
pd:
this is an exceptional case 😅